README.md

```diff
@@ -1,38 +1,67 @@
-## CosmWasm MultiTest README 
+# CosmWasm MultiTest 
+
+[![cw-multi-test on crates.io][crates-badge]][crates-url]
+[![docs][docs-badge]][docs-url]
+[![codecov][codecov-badge]][codecov-url]
+[![license][apache-badge]][apache-url]
+
+[crates-badge]: https://img.shields.io/crates/v/cw-multi-test.svg
+[crates-url]: https://crates.io/crates/cw-multi-test
+[docs-badge]: https://docs.rs/cw-multi-test/badge.svg
+[docs-url]: https://docs.rs/cw-multi-test
+[codecov-badge]: https://codecov.io/gh/CosmWasm/cw-multi-test/branch/main/graph/badge.svg?token=IYY72ZVS3X
+[codecov-url]: https://codecov.io/gh/CosmWasm/cw-multi-test
+[apache-badge]: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+[apache-url]: LICENSE
+[notice-url]: NOTICE
+
+> **Testing tools for multi-contract interactions**
 
 ## Introduction 
 
-CosmWasm `MultiTest` is a dynamic suite of testing tools designed for facilitating multi-contract 
-interactions within the CosmWasm ecosystem. Its primary focus is on providing developers
-with a robust framework for simulating complex contract interactions and bank operations. 
-Currently in the alpha stage, CosmWasm MultiTest is primarily used internally for testing cw-plus contracts. 
+**CosmWasm MultiTest** is a suite of testing tools designed for facilitating multi-contract 
+interactions within the [CosmWasm](https://github.com/CosmWasm) ecosystem.
+Its primary focus is on providing developers with a robust framework for simulating
+complex contract interactions and bank operations. Currently, **CosmWasm MultiTest**
+is in the _alpha_ stage, and primarily used internally for testing
+[cw-plus](https://github.com/CosmWasm/cw-plus) contracts. 
 
 ## Warning and Current Status 
-## Alpha Stage Caution 
 
-CosmWasm MultiTest is currently in its alpha development phase, primarily intended for internal use.
+### Alpha Stage Caution 
+
+**CosmWasm MultiTest** is currently in its alpha development phase, primarily intended for internal use.
```

Comment:
I think we can be more confident here now and don't call it alpha anymore

Reply:
Removed this subsection.